### PR TITLE
fix(pb): dont update immutable fields

### PR DESCRIPTION
**Pull request by Bladrak**
Fix https://github.com/ent/ent/issues/1427

### CODE REVIEW

File: entproto/cmd/protoc-gen-entgrpc/methods.go [hunk 1]
```diff
@@ -112,7 +112,7 @@ func (g *serviceGenerator) generateMutationMethod(op string) error {
 	}
 
 	for _, fld := range g.fieldMap.Fields() {
-		if fld.IsIDField {
+		if fld.IsIDField || (op == "update" && fld.EntField.Immutable) { // There's no Set* function for Immutable fields when updating
 			continue
 		}
```
Comment (a8m) [inline entproto/cmd/protoc-gen-entgrpc/methods.go:117]: Looks good to me. @rotemtam, let's change these string ops to `ent.Op` (e.g. `ent.UpdateOne`, `ent.Create`, etc). 
 
```suggestion
		if fld.IsIDField || (op == "update" && fld.EntField.Immutable) {
			continue
		}
```

**Review by a8m — APPROVED** [review]
@Bladrak, please re-run codegen (`go generate ./...`) and push the changes.

cc @rotemtam

File: entproto/cmd/protoc-gen-entgrpc/methods.go (hunk 1 above)
Comment (rotemtam) [reply entproto/cmd/protoc-gen-entgrpc/methods.go:117]: > @Bladrak, please re-run codegen (`go generate ./...`) and push the changes.
> 
> cc @rotemtam
> 
> Thanks a lot for your contribution btw 😃

more specifically @Bladrak, notice the CI breaks:
https://github.com/ent/contrib/pull/57/checks?check_run_id=2279944179
you will see there in the diff explanation:
```diff
-// 	protoc        v3.15.6
+// 	protoc        v3.15.5
```
If you bump the CI version here:
https://github.com/ent/contrib/blob/master/.github/workflows/ci.yml#L57
It should work

**Comment by Bladrak** [comment]
@a8m @rotemtam thanks for your suggestions, should be fine now, what do you think? :)

**Review by rotemtam — APPROVED** [review]
Perfect! Thanks a lot for your contribution @Bladrak.

**Pull request merged**